frontend/public/components/masthead-toolbar.jsx

```diff
@@ -360,7 +360,7 @@ class MastheadToolbar_ extends React.Component {
                 </Firehose>
             }
             {/* desktop -- (application launcher dropdown), help dropdown [documentation, about] */}
-            {flags[FLAGS.OPENSHIFT] && <ToolbarItem>
+            <ToolbarItem>
```

Comment:
We still need to flag the launch actions here.
This will also be overridden by changes in https://github.com/openshift/console/pull/1908
So maybe just leave it as is for now.